flybywiresim/a32nx
# fix(mcdu): on appr, show temp sign, remove fls/final app

Fixes #[issue_no]

## Summary of Changes

In accordance with new PERF APPR pages photos on an A320 with Honeywell FMS software revision 2, add hard sign to the temperature (this is different to A330 FMS revision 2 I previously used as reference). I also removed FINAL APP/FLS selection, pending us actually adding that feature (it is not shown in a real aircraft without the FLS feature).

I also noted that the title shows RNV rather than RNAV like us, but I'm not sure if that's specific to the loaded nav data so I didn't make that change.

## Screenshots (if necessary)


Before:
![image](https://user-images.githubusercontent.com/9995998/113666558-9080fa80-9703-11eb-8a68-e629a9fa2792.png)

After:
![image](https://user-images.githubusercontent.com/9995998/113791894-c2906c00-9798-11eb-84ec-fada5981bc48.png)
![image](https://user-images.githubusercontent.com/9995998/113792111-3df21d80-9799-11eb-9f6e-b597393fbdee.png)


## References

Photo on Discord from Tolip#6487.
![image](https://user-images.githubusercontent.com/9995998/113791223-4c3f3a00-9797-11eb-929e-e0b31388e48e.png)





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Check approach page temperature field with various inputs (range is [-99, 99])
- Check alignment of fields while altering values in all fields


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
